boost/teamcity_boost.cpp

```diff
@@ -71,7 +94,7 @@ struct TeamcityFormatterRegistrar {
     }
 };
 
-BOOST_GLOBAL_FIXTURE(TeamcityFormatterRegistrar)
+BOOST_GLOBAL_FIXTURE(TeamcityFormatterRegistrar);
 
 // Formatter implementation
 static std::string toString(boost::unit_test::const_string bstr) {
```

Comment:
I did try and go through your fork and add them to this patch (the ones that appeared to be relevant...like the conditional stuff and the current test name).  Maybe there are other parts to it that I have missed?